group/basic/main.tf

```diff
@@ -1,65 +0,0 @@
-# AWS Autoscaling group
-
-## Creates security group
-resource "aws_security_group" "sg_asg" {
-  description = "${var.stack_item_fullname} security group"
-  vpc_id      = "${var.vpc_id}"
-
-  tags {
-    Name        = "${var.stack_item_label}-asg-sg"
-    application = "${var.stack_item_fullname}"
-    managed_by  = "terraform"
-  }
-
-  egress {
```

Comment:
@blakeneyops ah didn't notice that this was pulled out of the module and requires the module users to define the egress rule (found it while bumping the asg version on the vpn module). I'll make a note of it in the breaking changes.